src/test/java/duke/utils/ParserTest.java

```diff
@@ -0,0 +1,67 @@
+package duke.utils;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import duke.exceptions.DukeException;
+import duke.commands.Command;
+import duke.commands.ByeCommand;
+
+import java.util.ArrayList;
+
+public class ParserTest {
+    @Test
+    public void testParse_Bye_ByeCommand() {
+        try {
```

Comment:
Don't think there is a need to put try catch statement in your tests. Just running an assert statement with the Junit library should be sufficient